pyscipopt/scip.pyx

```diff
@@ -362,31 +357,26 @@ cdef class Model:
         # http://docs.cython.org/src/reference/extension_types.html#finalization-dealloc
         PY_SCIP_CALL( SCIPfree(&self._scip) )
 
-    @scipErrorHandler
     def create(self):
         """Create a new SCIP instance"""
-        return SCIPcreate(&self._scip)
+        return PY_SCIP_CALL(SCIPcreate(&self._scip))
```

Comment:
Actually, you should also get rid of the `return`.

`PY_SCIP_CALL` doesn't return anything, but would raise an exception if necessary.